frontend/src/components/v2/Model/PagedTable.vue

```diff
@@ -66,6 +71,13 @@ const emit = defineEmits<{
   (event: "list:update", list: T[]): void;
 }>();
 
+const options = computed(() => {
+  return [50, 100, 200, 500, 1000].map((num) => ({
```

Comment:
Maybe unnecessary to be `1000` as it's already _paginated_.

Reply:
I can't get the point...Why not support 1000 items per page?

Reply:
> Why not support 1000 items per page?

Similar to why pagination is implemented:

1. Rendering large amounts of data may cause performance issues;
2. Users don’t need to, and can't, view all the data;

So I think if the page size is set to 1000 items, the above problems may still persist.

Reply:
It's okay for me to support 1000 or not, but 1000 is not a large number for pagination

Reply:
And our APIs support maximum 1000 or 5000 page size

Reply:
It's similar to SQL Editor results. We should have a meeting to discuss about this topic - scrolling, pagination, etc.

Reply:
Close for now